lerobot-web-fe/src/components/Teleoperate.tsx

```
@@ -86,23 +41,41 @@ export default function Teleoperate() {
+          {loading ? 'Loading...' : status || 'Not started'}
+          {pid && !loading ? ` – PID: ${pid}` : ''}
+        </p>
+        {error && <p className={styles.errorText}>⚠ Error: {error}</p>}
+      </div>
+
+      <button
+        className={`${styles.controlButton} ${isRunning ? styles.stop : styles.start}`}
+        onClick={isRunning ? handleStop : handleStart}
+        disabled={loading}
+      >
+        {false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)